runtime/sqldelight-runtime/native/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'konan'
+
+konan.targets = ['ios_arm64', 'ios_x64', 'macos']
+
+konanArtifacts {
+  library('sqldelight-runtime-native') {
+    enableMultiplatform true
+    artifactName 'SqlDelightRuntimeNative'
+  }
+
+  program('sqldelight-runtime-native-test', targets: ['macos']) {
+    enableMultiplatform true
+    srcDir 'src/test/kotlin'
+    commonSourceSet 'test'
+    libraries {
+      artifact 'sqldelight-runtime-native'
+    }
+    extraOpts '-tr'
+  }
+}
+
+dependencies {
+  expectedBy project(':runtime:sqldelight-runtime')
+}
```

Comment:
You're missing the `apply file` thing... although there's no Android or Java plugin so it will break and need updated to support konan.

Reply:
might use something like this instead: https://github.com/russhwolf/multiplatform-settings/blob/master/multiplatform-settings/build.gradle

Reply:
That's similar to what's in our current script except that we don't publish to Bintray. We should just fix our script to handle kotlin native.

Reply:
kk will do